components/apps.js

```diff
@@ -335,7 +335,7 @@ SteamUser.prototype.getProductInfo = function(apps, packages, inclTokens, callba
 							}
 
 							for (let packageid in packageTokens) {
-								tokenPackages.push({appid: parseInt(packageid, 10), access_token: packageTokens[appid]})
+								tokenPackages.push({appid: parseInt(packageid, 10), access_token: packageTokens[packageid]})
```

Comment:
Also `tokenPackages.push({packageid:`...

Reply:
Right! I thought it was `appid` in `tokenPackages`, too. But I went back to check the documents, and right, it should be `packageid`.
Thanks so much!